ui/component/core/src/index.ts

```diff
@@ -256,11 +263,10 @@ export class Manager implements EventProviderFactory {
         return this._config.mapType || MapType.VECTOR;
     }
 
+    protected static MAX_RECONNECT_DELAY = 45000;
```

Comment:
Just to double check; `WebsocketEventProvider` has a `MAX_RECONNECT_DELAY` field as well.
Maybe combining the default values, or reuse one of the variables?